test/model.js

```diff
@@ -580,6 +618,25 @@
     equal(newModel.destroy(), false);
   });
 
+  test("destroy will pass extra options to success callback", 1, function () {
+    var SpecialSyncModel = Backbone.Model.extend({
+      sync: function (method, model, options) {
+        _.extend(options, { specialSync: true });
+        return Backbone.Model.prototype.sync.call(this, method, model, options);
+      },
+      urlRoot: '/test'
+    });
+
+    var model = new SpecialSyncModel({ id: 'id' });
+
+    model.destroy({ success: onSuccess });
+    this.ajaxSettings.success();
+
+    function onSuccess(model, response, options) {
```

Comment:
No I understand what you're saying, I just disagree it reads easier ;)

Thanks for fixing that up.